packages/react-ui/components/Loader/Loader.tsx

```diff
@@ -253,4 +270,49 @@ export class Loader extends React.Component<LoaderProps, LoaderState> {
       spinnerStyle,
     });
   };
+
+  private disableChildrenFocus() {
+    if (!this.childrenObserver) {
+      this.makeObservable();
+    }
+    if (!this.childrenContainerNode) {
+      return;
+    }
+    const tabbableElements = tabbable(this.childrenContainerNode);
+    tabbableElements.forEach((el) => {
+      if (!el.hasAttribute('origin-tabindex')) {
+        el.setAttribute('origin-tabindex', el.tabIndex + '');
+      }
+      el.tabIndex = -1;
+    });
+  }
+
+  private enableChildrenFocus() {
+    this.makeUnobservable();
+    document.querySelectorAll('[origin-tabindex]').forEach((el) => {
+      el.setAttribute('tabindex', el.getAttribute('origin-tabindex') ?? '0');
+      el.removeAttribute('origin-tabindex');
+    });
+  }
+
+  private makeObservable() {
+    const target = this.childrenContainerNode;
+    if (!target) {
+      return;
+    }
+    const config = {
+      childList: true,
+      subtree: true,
+    };
+    const observer = new MutationObserver(() => this.toggleChildrenFocus());
```

Comment:
```suggestion
    const observer = new MutationObserver(this.disableChildrenFocus);
```

На самом деле выходит что тут только disable уже актуален.